src/ripple/app/paths/PathRequest.cpp

```diff
@@ -307,8 +307,7 @@ int PathRequest::parseJson (Json::Value const& jvParams)
         return PFR_PJ_INVALID;
     }
 
-    convert_all_ =
-        saDstAmount == STAmount(saDstAmount.issue(), 1u, 0, true);
+    convert_all_ = saDstAmount == STAmount(saDstAmount.issue(), 1u, 0, true);
```

Comment:
I can't believe you didn't change that variable name!  :-(

Couldn't I get you to change it now!?  Variables are the one thing we're mostly consistent in.

And we STILL don't have a function that says if something is `1` so we still construct a fairly expensive `STAmount` just to check for "oneness".  Can we please finally fix this?